startlxqtwayland.in

```diff
@@ -140,6 +140,15 @@ elif [ "$COMPOSITOR" = "kwin_wayland" ]; then
     # WARNING: Option '--no-kactivities' can result in crashes with animations and corner actions.
     exec ${COMPOSITOR}_wrapper --exit-with-session lxqt-session --xwayland
 
+elif [ "$COMPOSITOR" = "miriway" ]; then
+    export XDG_CURRENT_DESKTOP="LXQt:$COMPOSITOR:mir"
+    if [ ! -f "$XDG_CONFIG_HOME/lxqt/miriway-shell.config" ]; then
+        cp "$share_dir"/lxqt/wayland/miriway/miriway-shell.config "$XDG_CONFIG_HOME/lxqt/miriway-shell.config"
+       if echo "$valid_layouts" | grep -q "$trylayout"; then
+          echo "keymap=$trylayout" >> "$XDG_CONFIG_HOME/lxqt/miriway-shell.config"
+       fi
```

Comment:
You're missing an `fi` here on line 150, it throws an error in the rpmlint checks, when building on openSUSE

Reply:
fixed